README.md

```diff
@@ -1,6 +1,6 @@
 # JHipster Online
 
-[![Azure DevOps Build Status][azure-devops-image]][azure-devops-url-main]  [![Build Status][travis-image]][travis-url]  [![Docker Pulls](https://img.shields.io/docker/pulls/jhipster/jhipster-online.svg)](https://hub.docker.com/r/jhipster/jhipster-online/)
+[![Azure DevOps Build Status][azure-devops-image]][azure-devops-url-main] [![Build Status][travis-image]][travis-url] [![Docker Pulls](https://img.shields.io/docker/pulls/jhipster/jhipster-online.svg)](https://hub.docker.com/r/jhipster/jhipster-online/)
```

Comment:
I also do not understand the reason to have a Travis build badge. The last Travis build seems long time ago and I think we no longer use Travis. So suggest removing that badge as well and instead adding a GitHub Actions badge.

Reply:
Done at, https://github.com/jhipster/jhipster-online/pull/168